src/coreclr/jit/lclvars.cpp

```diff
@@ -8286,6 +8287,13 @@ Compiler::fgWalkResult Compiler::lvaStressLclFldCB(GenTree** pTree, fgWalkData*
 
     if (bFirstPass)
     {
+        // Ignore locals that alreay have field appearances
```

Comment:
typo
```suggestion
        // Ignore locals that already have field appearances
```

Reply:
My spelling seems to be regressing recently. 

Hate to have to retrigger CI just to fix things like this.